src/App.js

```diff
@@ -22,6 +22,19 @@ const useStyles = makeStyles((theme) => ({
   selectEmpty: {
     marginTop: theme.spacing(2),
   },
+  footer : {
+    ['@media (min-width:326px)']: {
+      position: "fixed",
+    },
+    ['@media (min-height:580px)']: {
+      bottom: "25px",
+    },
```

Comment:
```suggestion
    '@media (min-width:326px)': {
      position: "fixed",
    },
    '@media (min-height:580px)': {
      bottom: "25px",
    },
```

Two things: 
- We don't need to dynamically calculate the keys
- Run `npx prettier` once?

Reply:
Need elaboration on point 1